test/e2e/upgrade_test.go

```diff
@@ -74,7 +74,7 @@ func backup_upgrade_restore_test(useVolumeSnapshots bool) {
 		if installVelero {
 			//Set veleroImage and resticHelperImage to blank
 			//veleroImage and resticHelperImage should be the default value in originalCli
-			Expect(veleroInstall(context.Background(), upgradeFromVeleroCLI, "", "", veleroNamespace, cloudProvider, objectStoreProvider, useVolumeSnapshots,
+			Expect(veleroInstall(context.Background(), upgradeFromVeleroCLI, "", "", Plugins, veleroNamespace, cloudProvider, objectStoreProvider, useVolumeSnapshots,
```

Comment:
Should not pass the parameter `plugins` when installing the old version of cli

Reply:
As the old version cli is always a released one, we should pass null here to make it choose the plugins based on version

Reply:
It's a good catch, thanks!